tests/by-util/test_tee.rs

```diff
@@ -77,7 +77,7 @@ fn test_tee_no_more_writeable_1() {
     // equals to 'tee /dev/full out2 <multi_read' call
     let (at, mut ucmd) = at_and_ucmd!();
     let content = (1..=10).fold(String::new(), |mut output, x| {
-        let _ = writeln!(output, "{x}");
+        writeln!(output, "{x}").unwrap();
```

Comment:
Did this really fail before? Writing to a stribg is usually fine. We could even rewrite this with `push_str`, I think, which doesn't return a result.

Reply:
This didn't fail before, you're right! It's just weird to drop a `Result`. If this ever fails, we definitely want the test to fail, hence the `unwrap`.